cmd/publishing-bot/config/config.go

```diff
@@ -18,6 +18,14 @@ package config
 
 // Config is how we are configured to talk to github.
 type Config struct {
+	// GithubHost is the address for github.
+	// Default to github.com
+	GithubHost string `yaml:"github-host"`
+
+	// BasePackage is the base package name for this repo.
+	// Default to k8s.io
```

Comment:
only for SourceOrg kubernetes. Otherwise, it is github.com/SourceOrg, isn't it?

Reply:
correct!